### PR TITLE
[DEVENGAGE-2164] - Blog - AnswerCall and MakeCall via API

### PR DESCRIPTION
**Pull request by themooks**

### CODE REVIEW

File: source/2023-09-28-answer-call-via-api/index.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+Answering a call or placing a call using Platform API **is fully supported** (*"Maintain Persistent Connection" setting can be Enabled or Disabled*).  
+This is applicable to Genesys Cloud Embeddable Framework and to Genesys Cloud Web/Desktop app.
+
+If you are using Genesys Cloud Embeddable Framework, the use of `Interaction.updateState(action=pickup)` and `clickToDial(type=call)` actions **is fully supported** as well (*"Maintain Persistent Connection" setting can be Enabled or Disabled*).
+
+
+### Using Remote Phones or Generic SIP Phones (no remote answer support)
+
+Answering a call using Platform API **is NOT supported** (to be more specific, it is only supported while the phone is on an active persistent connection - the request will not be processed otherwise).
```
Comment (carnellj-genesys) [inline source/2023-09-28-answer-call-via-api/index.md:211]: Nice.

File: source/2023-09-28-answer-call-via-api/index.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+## Closing Thoughts
+
+You should now have the necessary information to understand when a call can be answered or initiated using the Platform API, and when it is not possible.
+
+Thank you and have fun!
+
+---
+---
+
+## Appendix A: Sample "*ForceDisconnect*" Architect Inbound Call flow
```
Comment (carnellj-genesys) [inline source/2023-09-28-answer-call-via-api/index.md:228]: Do we have a blueprint or accelerator around this?  This sounds like a really good accelerator to have.  Might want to talk to @AgnesCorpuz and see if we should open a separate ticket to create this content.  Really like the flow.  Also do we want to include the sample YAML for the flow here.

**Review by carnellj-genesys — CHANGES_REQUESTED** [review]
Overall really good blog post. Let me know your thoughts on feedback

**Comment by themooks** [comment]
Thanks. I think I have taken into account all feedback. Pushing an update now

**Review by carnellj-genesys — APPROVED** [review]

**Pull request merged**